test/inductor/test_profiler.py

```diff
@@ -75,6 +75,7 @@ def _test_profiling_kernel_names(self, fn, args, kernel_name_str: str):
                     hasattr(event, "name")
                     and kernel_name_str in event.name
                     and "triton" in event.name
+                    and "OriginOps" in event.name
```

Comment:
Can you add a test that does `export_chrome_trace` on a trace that has OriginOps in it? Reason: the json serialization code is a bit buggy, so it would be good to have a sanity check that the OriginOps format doesn't expose any serialization bugs

Reply:
done.
Added **test_profiling_origin_ops_with_chrome_json**